common/components/WalletDecrypt/Trezor.jsx

```diff
@@ -1,26 +1,139 @@
+// @flow
+import './Trezor.scss';
 import React, { Component } from 'react';
 import translate from 'translations';
+import TrezorConnect from 'vendor/trezor-connect';
+import DeterministicWalletsModal from './DeterministicWalletsModal';
+import TrezorWallet from 'libs/wallet/trezor';
+
+/* eslint-disable quotes */
+const TREZOR_PATHS = [
```

Comment:
consider moving out of component (and into config)

Reply:
👍